dbt_subprojects/dex/models/_projects/zeroex/zeroex_api_fills_deduped.sql

```diff
@@ -13,20 +13,47 @@
   ref('zeroex_arbitrum_api_fills_deduped')
   ,ref('zeroex_avalanche_c_api_fills_deduped')
   ,ref('zeroex_base_api_fills_deduped')
-  ,ref('zeroex_celo_api_fills_deduped')
   ,ref('zeroex_ethereum_api_fills_deduped')
-  ,ref('zeroex_fantom_api_fills_deduped')
```

Comment:
removed celo due to failing unique_combination_of_columns test. will revise and submit new pr

Reply:
re-added without failing unique_cols test after adding version number in `zeroex_fantom_api_fills_deduped`